### ISSUE TITLE
Every module fails to load with Error 1114 in MAX/MSP

### ISSUE THREAD

**Issue by merhard75** [CNMAT/CNMAT-Externs#126]
I follwod the instruction to compile the package with cygwin etc on windows 10, the build itself worked, but every attempt to use any object of this package failes with Error 1114 in Max/Map 8.15. How can I debug this or even solve that problem?

**Comment by wjslager** [comment]
I'm having the same issue. Did you manage to fix it? I got as far as googling "1114 loading issue" and finding out its probably something related to loading the DLL. Could be fixed by changing some GPU power settings but I have not managed it yet.

**Comment by merhard75** [comment]
nope I wasn't able to fix it :/

**Comment by equilet** [comment]
we are currently investigating the issue.

**Comment by carlesanton** [comment]
Any updates on it? I am facing the same problem. I also got 215 compilation errors on the `src\midifile\midifile.java`, most of them `error: cannot find symbol` for ` Atom`, I don't know if this may be related, but I have the same error with `odot` package and I got no compilation errors there.

Thank you

**Comment by equilet** [comment]
yes we have a fix iin place and it will be rolled into an upcoming release.  Feel free to ping me directly for a copy.
